src/components/application_manager/src/resumption/resume_ctrl.cc

```diff
@@ -658,7 +658,9 @@ bool ResumeCtrl::CheckAppRestrictions(
   LOG4CXX_AUTO_TRACE(logger_);
   DCHECK_OR_RETURN(saved_app.keyExists(strings::hmi_level), false);
 
+#ifdef ENABLE_LOG
   const bool is_media_app = application->is_media_application();
+#endif
```

Comment:
You should be able to simply remove this local variable and change the log line so that it doesn't use the variable:
```cpp
  LOG4CXX_DEBUG(logger_,
                "is_media_app " << application->is_media_application()
                                << "; hmi_level " << hmi_level << " result "
                                << result);
```